test/components/layouts/indexlayout.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { shallow } from "enzyme";
+import IndexLayout from "./../../../src/layouts/IndexLayout";
+import Link from "next/link";
+
+describe("Layout Component Tests", () => {
```

Comment:
Nitpick: I don't think the word "tests" adds anything here...could we change it to just `describe("Layout component", ...` or `describe("IndexLayout", ...`?